src/webrtc/call.ts

```diff
@@ -631,19 +631,17 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
         if (!this.client.getUseE2eForGroupCall()) return;
         // It's possible to want E2EE and yet not have the means to manage E2EE
         // ourselves (for example if the client is a RoomWidgetClient)
-        if (!this.client.isCryptoEnabled()) {
+        if (!this.client.getCrypto()) {
             // All we know is the device ID
             this.opponentDeviceInfo = new DeviceInfo(this.opponentDeviceId);
             return;
         }
-        // if we've got to this point, we do want to init crypto, so throw if we can't
-        if (!this.client.crypto) throw new Error("Crypto is not initialised.");
-
         const userId = this.invitee || this.getOpponentMember()?.userId;
 
         if (!userId) throw new Error("Couldn't find opponent user ID to init crypto");
 
-        const deviceInfoMap = await this.client.crypto.deviceList.downloadKeys([userId], false);
+        // Here we were calling `MatrixClient.crypto.deviceList.downloadKeys` which is not supported by the rust cryptography.
+        const deviceInfoMap = new Map();
```

Comment:
The method isn't deprecated - can we just continue using it and wait for Rust crypto to implement it?

Reply:
Oh I see, it's DeviceInfo that's been replaced. The chances we want to resurrect this code are indeed low so I guess marking this as broken isn't the end of the world.

Reply:
The `crypto` property of `MatrixClient` is deprecated and has `undefined` value when the rust crypto stack is used.